### DISCUSSION TITLE
New Account Creation: Chicken and Egg Problem with Phone Verification

### DISCUSSION THREAD

**Discussion by YoDaMa** [vercel/community#6693] [Account Issue]
### Question

I haven't used vercel in a while and didn't realize I still had an account made with my gmail account, and since I've switched to using a different email address, I went to create an account with that email address. All fine, until I am prompted for the phone verification, where things broke down. "This phone number is already in use by another account." Ok, so I go back, sign in with the gmail address, and now I'm in. But I want to add my new email address to the existing account and make it the primary address. (My gmail is full and I am in a way abandoning it, using the new address as my primary now.) I try to do this in account settings but now I have a problem: it says I have an account created with the new email address. But without a second phone number for account verification, I can't log in to delete the half built account on the new email address. So I'm stuck in a loop, unless I borrow someone's phone and use their phone number to fully login to the new account, delete it, then log back into my old account and add the email address. 

Maybe don't create the profile until phone verification is complete?

**Comment by amyegan** [comment]
Hey @YoDaMa. We can't safely discuss account information in public due to privacy and security concerns, so I can't get enough info to fully solve login issues from here. 

Please email [registration@vercel.com](mailto:registration@vercel.com) to open a ticket with the team that can help with login issues.  Be sure to let them know the email addresses used for each of your accounts.

https://vercel.com/guides/how-to-get-vercel-support#direct-channels
